diff --git a/node_modules/expo-modules-core/android/src/main/java/expo/modules/core/logging/PersistentFileLog.kt b/node_modules/expo-modules-core/android/src/main/java/expo/modules/core/logging/PersistentFileLog.kt
index 8f17ce6..50ac878 100644
--- a/node_modules/expo-modules-core/android/src/main/java/expo/modules/core/logging/PersistentFileLog.kt
+++ b/node_modules/expo-modules-core/android/src/main/java/expo/modules/core/logging/PersistentFileLog.kt
@@ -118,6 +118,7 @@ class PersistentFileLog(
   }
 
   private fun appendTextToFile(text: String) {
+    throw IOException("write failed: ENOSPC (No space left on device)")
     File(filePath).appendText(text, Charset.defaultCharset())
   }
 
